src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -49,15 +52,18 @@ public BtConnectorHelper() {
         this.context = jxcore.activity.getBaseContext();
     }
 
-
     public BTConnector.WifiBtStatus Start(String peerName,int port){
         this.mServerPort = port;
         this.myPeerIdentifier= GetBluetoothAddress();
         this.myPeerName = peerName;
         this.lastAvailableList.clear();
+
         Stop();
-        mBTConnector = new BTConnector(context,this,this,conSettings);
-        return mBTConnector.Start(this.myPeerIdentifier,this.myPeerName);
+
+        BTConnector tmpCon= new BTConnector(context,this,this,conSettings);
+        BTConnector.WifiBtStatus  ret = tmpCon.Start(this.myPeerIdentifier,this.myPeerName);
+        mBTConnector = tmpCon;
+        return ret;
     }
 
     public boolean isRunning(){
```

Comment:
And yes (I just saw this in IntelliJ as well), this can be simplified to `return mBTConnector != null`